src/Middleware/Spa/SpaProxy/src/SpaHostingStartup.cs

```diff
@@ -19,6 +21,9 @@ public void Configure(IWebHostBuilder builder)
                 if (File.Exists(Path.Combine(AppContext.BaseDirectory, "spa.proxy.json")))
                 {
                     services.AddHostedService<SpaProxyLaunchManager>();
+                    services.AddSingleton<SpaProxyStatus>();
+                    services.TryAddEnumerable(ServiceDescriptor.Singleton<IConfigureOptions<SpaDevelopmentServerOptions>, ConfigureSpaDevelopmentServerOptions>());
```

Comment:
Can consider just doing the IConfigure inline here to avoid the class:

something like this would let you avoid the internal class and simplify the bind using the configuration overload which binds implicitly
```
            var configuration = new ConfigurationBuilder()
                .AddJsonFile(Path.Combine(AppContext.BaseDirectory, "spa.proxy.json"))
                .Build();
            services.Configure<SpaDevelopmentServerOptions>(configuration.GetSection("SpaProxyServer");
```